profile/src/hierarchical_deterministic/cap26/cap26_path/cap26_path.rs

```diff
@@ -0,0 +1,215 @@
+use crate::prelude::*;
+
+/// A derivation path design specifically for Radix Babylon wallets used by Accounts and Personas
+/// to be unique per network with separate key spaces for Accounts/Identities (Personas) and key
+/// kind: sign transaction or sign auth.
+#[derive(
+    Clone,
+    Debug,
+    PartialEq,
+    EnumAsInner,
+    Eq,
+    Hash,
+    PartialOrd,
+    Ord,
+    SerializeDisplay,
+    DeserializeFromStr,
+    derive_more::Display,
+    uniffi::Enum,
+)]
+pub enum CAP26Path {
```

Comment:
`derive_more` crate did unfortunately not seem to support using `#[display(..)]` attached to the enum itself :/